telemetry/roles/prometheus_gaudi/main.yml

```diff
@@ -0,0 +1,2 @@
+- name: Deploy Gaudi metric exporter
```

Comment:
missing copyright message

Reply:
this file should be within tasks folder

Reply:
Would add the copyright and move it to tasks folder.
And would add a variable prometheus_gaudi with default value false, is it ok?

Reply:
prometheus_gaudi_support will be ok and default should be false